### PR TITLE
HRT-28 create course info

### PR DESCRIPTION
**Pull request by EricPedrido**
### Purpose

Adds front end for creating chart/providing course information just before starting the game.

[HRT-28](https://softeng761team5.atlassian.net/browse/HRT-28)

### Approach

- Adds a new page for creating course information
- Form is responsive to window size changes
- Added antd icons to the package. Not sure if it is added in the PR, but if you need to, then it's: 
```javascript 
npm install --save @ant-design/icons
```

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

#### Does this PR involve a UI change?

- [X] Yes
- [ ] No

#### Screenshot(s) of UI before this PR
n/a

#### Screenshot(s) of UI after this PR
screenshot
)

### CODE REVIEW

**Review by lucycy98 — CHANGES_REQUESTED** [review]
hiii looks good 2 things: 
- can you please rebase with master? its a little out of date. 
- are we putting shadows for all page titles? e.g 'create chart'
- when I select an item for course age and cohort size, the text field should be filled with the value I select. currently it is just 'select'. see below:
image

**Comment by EricPedrido** [comment]
> * can you please rebase with master? its a little out of date.

Yup will do

> * are we putting shadows for all page titles? e.g 'create chart'

This is how it was in the hifi prototypes, but i can remove the shadows for consistency with other pages

> * when I select an item for course age and cohort size, the text field should be filled with the value I select. currently it is just 'select'.

I will need to look into this, I've done it how they've documented on antd's website but I'll try find a fix somewhere online

File: src/pages/CourseInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+import Header from "../components/Header";
+import "../styles/CourseInfo.css";
+
+const { Option } = Select;
+
+const CourseInfo: React.FC = () => {
+  const onConfirmClick = () => {
+    // TODO: Write code here to redirect to cards screen
+  };
```
Comment (preetpatel) [inline src/pages/CourseInfo.tsx:13]: Think the cards endpoint is now available :)

File: src/pages/CourseInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+  const onCancelClick = () => {
+    // TODO: Write code here to redirect back to homepage
```
Comment (preetpatel) [inline src/pages/CourseInfo.tsx:16]: This can also be implemented now

**Review by lucycy98 — APPROVED** [review]
thanks for the changes - LGTM

**Review by mitchfaulcon — APPROVED** [review]
Looks good to me, I'll implement the page routing when I do the play reason ticket

**Pull request merged**